scripts/install_requirements.sh

```diff
@@ -3,4 +3,5 @@
 pip install -r requirements.txt
 python -m nltk.downloader punkt
 python -m spacy.en.download all
-pip install --no-cache-dir http://download.pytorch.org/whl/cu80/torch-0.1.11.post5-cp35-cp35m-linux_x86_64.whl
\ No newline at end of file
+pip install --no-cache-dir http://download.pytorch.org/whl/cu80/torch-0.1.11.post5-cp35-cp35m-linux_x86_64.whl
+conda install pytorch torchvision -c soumith
```

Comment:
When you install conda, it puts a new pip binary on your path.  I use the conda version of pip.  That is to say, you should be able to replace pip with conda everywhere, and it should be fine.  There isn't any particular reason that we used pip originally in those scripts - feel free to make it consistent, if it works.